folsom/src/main/java/com/spotify/folsom/reconnect/ReconnectingClient.java

```diff
@@ -75,6 +76,40 @@ public ReconnectingClient(
       final int maxSetLength,
       final EventLoopGroup eventLoopGroup,
       final Class<? extends Channel> channelClass) {
+    this(
+        backoffFunction,
+        scheduledExecutorService,
+        address,
+        outstandingRequestLimit,
+        eventLoopThreadFlushMaxBatchSize,
+        binary,
+        authenticator,
+        executor,
+        connectionTimeoutMillis,
+        charset,
+        metrics,
+        maxSetLength,
+        eventLoopGroup,
+        channelClass,
+        null);
+  }
+
+  public ReconnectingClient(
```

Comment:
Does this need to be public?

Reply:
The constructor is called by MemcacheBuilderClient which is in a different package, so I think it needs to be public.